FX2024 Domain & Capabilities update / Understand cloud usage and cost (#642)
microsoft/finops-toolkit: docs/_docs/framework/capabilities/understand/reporting.md

Understanding your cost data is critical to enable accurate and meaningful showback to all stakeholders.

> If there is the need to report from multiple sources, for example multiple cloud providers, configuration management database (CMDB), or  IT asset management (ITAM) systems, consider using the FinOps Open Cost & Usage Specification ([FOCUS project](https://finops.org/focus)) as the standard billing schema for this report. To learn why organizations needs it and why Microsoft belives in FOCUS, you can review the [FOCUS documentation](https://microsoft.github.io/finops-toolkit/focus) available on FinOps toolkit provided by Microsoft.

Reviewer: ```suggestion
If there is the need to report from multiple sources, for example multiple cloud providers, configuration management database (CMDB), or  IT asset management (ITAM) systems, consider using the FinOps Open Cost & Usage Specification ([FOCUS project](../../../focus/README.md)) as the standard billing schema for this report.
```

- Use the different tools optimized to provide the details you need to understand the holistic picture of your resource cost and usage.
  - [Analyze resource usage metrics in Azure Monitor](https://learn.microsoft.com/azure/azure-monitor/essentials/tutorial-metrics).
  - [Review resource configuration changes in Azure Resource Graph](https://learn.microsoft.com/azure/governance/resource-graph/how-to/get-resource-changes).

> - If you need to build more advanced reports or merge cost data with other cloud or business data, [leverage the FinOps toolkit Power BI reports](../../../../_reporting/power-bi/README.md) part of the open source FinOps toolkit provided by Microsoft..

Reviewer: ```suggestion
- If you need to build more advanced reports or merge cost data with other cloud or business data, [leverage the FinOps toolkit Power BI reports](../../../../_reporting/power-bi/README.md) part of the open source FinOps toolkit provided by Microsoft.
```

    - FinOps hubs will accelerate your development and help you focus on building the features you need rather than infrastructure.

>     - FinOps hubs includes a [Power BI report](https://microsoft.github.io/finops-toolkit/power-bi) that normalizes data to the FOCUS schema, which can be a good starting point.

Reviewer: ```suggestion
    - FinOps hubs includes a [Power BI report](../../../../reporting/power-bi) that normalizes data to the FOCUS schema, which can be a good starting point.
```

## 🍎 Learn more at the FinOps Foundation

> This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Reporting + analytics capability](https://www.finops.org/framework/capabilities/???/) article in the FinOps Framework documentation.

Reviewer: ```suggestion
This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Reporting and analytics capability](https://www.finops.org/framework/capabilities/reporting-analytics/) article in the FinOps Framework documentation.
```

You can also find related videos on the FinOps Foundation YouTube channel:

<!--[!VIDEO https://www.youtube.com/embed/{id}?list={list}]-->

> {% include video.html title="Reporting + analytics videos" id="CVTJLdcozj1eEpxT" list="PLUSCToibAswlDSQdehKhi7ysP2hmetigl" %}

Reviewer: ```suggestion
{% include video.html title="Reporting and analytics videos" id="CVTJLdcozj1eEpxT" list="PLUSCToibAswlDSQdehKhi7ysP2hmetigl" %}
```
